spec/core_functions/color/is_powerless.hrx

```diff
@@ -0,0 +1,77 @@
+<===> no_space/input.scss
+a {b: is-powerless('red', 'hue')}
+
+<===> no_space/output.scss
```

Comment:
```suggestion
<===> no_space/output.css
```
Here and throughout this file, the output files should end with `.css`

Reply:
@oluoluoxenfree All of the output files should end with `.css`. The tests are taking the input, compiling it to CSS and comparing the results. If the the output file doesn't end in `.css` (error files follow a different format) the test runner will throw an error and say that the `output` file is missing.